wandb/sdk/wandb_artifacts.py

```diff
@@ -74,6 +74,8 @@
 
 ARTIFACT_TMP = tempfile.TemporaryDirectory("wandb-artifacts")
 
+DISABLE_WRITE_CACHE = "WANDB_ARTIFACTS_DISABLE_WRITE_CACHE"
```

Comment:
did you consider making this part of W&B settings? I am not sure if the setting object is available in the contexts you need it, but if it is, it would be great to use the existing settings patterns as opposed to a one-off env var